packages/core/admin/admin/src/pages/SettingsPage/components/Tokens/DeleteTokenDialog/tests/DeleteTokenDialog.test.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { act, fireEvent, render } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { NotificationsProvider } from '@strapi/helper-plugin';
+
+import DeleteTokenDialog from '../index';
+
+function ComponentToTest(props) {
+  return (
+    <IntlProvider locale="en" messages={{}}>
+      <ThemeProvider theme={lightTheme}>
+        <NotificationsProvider toggleNotification={() => {}}>
+          <DeleteTokenDialog onClose={() => {}} onConfirm={() => {}} isOpen {...props} />
+        </NotificationsProvider>
+      </ThemeProvider>
+    </IntlProvider>
+  );
+}
+
+const setup = (props = { onClose: jest.fn(), onConfirm: jest.fn() }) => {
+  return render(<ComponentToTest {...props} />);
+};
+
+describe('DeleteTokenDialog', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  it('renders and matches the snapshot', () => {
+    const { baseElement } = setup();
+    expect(baseElement).toMatchSnapshot();
```

Comment:
Do you think this test is beneficial?

Reply:
I have added a test for the DeleteButton